service/openapi/src/test/java/org/dotwebstack/framework/service/openapi/handler/CoreRequestHandlerTest.java

```diff
@@ -212,6 +214,41 @@ public void getResponseTest()
         .is2xxSuccessful();
   }
 
+  @SuppressWarnings("unchecked")
+  @Test
+  public void getRedirectResponseTest() throws Exception {
+    MultiValueMap<String, String> queryParams = new LinkedMultiValueMap<>();
+    ServerRequest request = mock(ServerRequest.class);
+    when(request.queryParams()).thenReturn(queryParams);
+
+    Map<Object, Object> data = new HashMap<>();
+    data.put("data", "{\"key\" : \"value\" }");
+
+    ServerRequest.Headers headers = mock(ServerRequest.Headers.class);
+    HttpHeaders asHeaders = mock(HttpHeaders.class);
+    when(headers.asHttpHeaders()).thenReturn(asHeaders);
+    when(request.headers()).thenReturn(headers);
+
+    Mono<String> mono = mock(Mono.class);
```

Comment:
Ik denk dat deze niet gemocked hoeft te worden:
```suggestion
 Mono<String> mono = Mono.just(null);
    when(request.bodyToMono(String.class)).thenReturn(mono);
```

Reply:
aangepast met een .empty()